timemachine/fe/mle.py

```diff
@@ -70,37 +70,139 @@ def wrapped(x):
     return wrapped
 
 
-def infer_node_vals(edge_idxs, edge_diffs, edge_stddevs):
+def infer_node_vals(edge_idxs, edge_diffs, edge_stddevs, ref_node_idxs=tuple(), ref_node_vals=tuple()):
     """
     Given pairwise comparisons involving K states,
     return a length-K vector of underlying absolute values
     by finding node_vals that maximizing likelihood of edge_diffs.
-    The result is determined up to an arbitrary additive constant --
-    by convention this function sets result[0] = 0.0.
+
+    Reference node vals influence result via a single additive offset.
 
     Parameters
     ----------
     edge_idxs: [E, 2] array, where the values are in range(K).
     edge_diffs: [E] array, relative values
     edge_stddevs: [E] array, positive
 
+    ref_node_idxs: [N_ref] int array
+    ref_node_vals: [N_ref] array
+
+    TODO: incorporate ref_node_stddevs here?
```

Comment:
This should probably be made an inline comment rather then part of the parameters docstring